generate-stackbrew-library.sh

```diff
@@ -94,7 +94,7 @@ for version in "${versions[@]}"; do
 		variantArches=( amd64 arm32v5 arm32v7 arm64v8 i386 mips64le ppc64le s390x)
 
 		case "$version" in
-		    23|22|21|20|19|18)
+		    25|24|23|22|21|20|19|18)
```

Comment:
seems like you are adding? this a filter that prevent those version to get any other arch?